docs/usage.md

```diff
@@ -101,14 +101,14 @@ In this case, there is no `match` key, so only one rule needs to match:
 `rules` is an array of rules that can be used to target specific files with your defined syntax file.  The rules are processed until the first rule matches, so order your rules in a way that makes sense to you.
 
 ### Filename Rule
-Filename rule defines a filename with regex.  This uses the full file path.
+Filename rule defines a filename with regex.  The pattern is matched against the beginning of the full file path (there is an implicit `^`).
 
 ```js
 {"file_name": ".*\\.xml(\\.dist)?$"},
 ```
 
 ### First Line Rule
-First line rule allows you to check if the first line of the files content matches a given regex.
+First line rule allows you to check if the first line of the files content matches a given regex. The pattern is matched against the beginning of the line (there is an implicit `^`).
```

Comment:
Again, the example below is showing an explicit `^`, so this isn't really a true statement...though yeah, you can probably get away with an implicit `^`.